src/ngraph/frontend/onnx_import/graph.cpp

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "graph.hpp"
+#include "node.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        Graph::Graph(const onnx::GraphProto& graph_proto)
```

Comment:
I think using `ngraph::onnx_import::Graph::Graph` breaks the way CLion is able to analyze and navigate the code. Using this structure helps our tools.

Reply:
Correct, we need this to efficiently work with a large code in the tools we use. For example, it is much easier to fold the whole block of code at once instead of folding each line separately. And not all the time the code needs to be visible since there're IntelliSense things build in various tools.